cli/src/utils/loadLocalCommands.ts

```diff
@@ -229,6 +231,16 @@ async function runCommand(
 
   params["_arguments"] = _arguments;
 
+  for (const [key, inputOpts] of Object.entries(instance.inputs)) {
+    if (typeof inputOpts.formatter === "function") {
+      params[key] = await inputOpts.formatter(params[key]);
+    }
+
+    if (typeof inputOpts.validator === "function") {
+      await inputOpts.validator(params[key]);
+    }
+  }
```

Comment:
From https://github.com/actionhero/actionhero/pull/2062

Reply:
Gave it a (very quick) shot, but there's a comment in our `loadLocalCommands` file that says we don't want to require `actionhero` in the `cli` package, which makes this tricky...

Reply:
That's in the `getActionhero` method itself... I wonder if it is OK to use that method to load a method like

```
function doStuff (...args) {
  const originalDoStuff = getActionhero()('doStuff')
  return originalDoStuff(args...)
}
```

Reply:
Ah I see! That doesn't seem _terrible_ (aside from not getting typings)

Looks like some work needs to be done on the actionhero side though - these cli methods are being defined from within the actionhero cli (and not exported to actionhero). Grouparoo also passes around the commander `program`. This probably means refactoring a bit to move these methods into some utility module that can be exported, taking in which `program` to use.